### ISSUE TITLE
doubts

### ISSUE THREAD

**Issue by vaishnavm217** [lxg2015/faceboxes#4]
Hi,
I made changes to your network. Added the batchnorm and xavier initializations, but i noticed you have used Adam optimizer while paper used SGD with decay and momentum, even the parameters are not the same of paper. I followed the same methodology, but my loss is still pretty high (~3.5),still need to do eval using widerface eval.
1. Did you train it on widerface train+val or only train?
2. For eval, did you use the evaluation tool present in site?

**Comment by Vermeille** [comment]
Yeah the code is reaaaaaaaally weird...

- The number of filters in the first layer is really low
- No BN, beside CReLU, _not a single non-linearity. Look at the Inception cells, they have no ReLU. What are they worth then?
- The paper talks about a "Scale" layer in the CReLU layer, that's nowhere to be found here.

If you've found some explanations, I'll gladly listen to them
